test/integration/scan_test.go

```diff
@@ -1167,6 +1168,41 @@ func TestScanCreateWithSSHKey(t *testing.T) {
 	executeCmdWithTimeOutNilAssertion(t, "Create a scan with ssh-key should pass", 4*time.Minute, args...)
 }
 
+func TestScanGLReportValidation(t *testing.T) {
+	_, projectName := getRootProject(t)
```

Comment:
Use GenerateRandomProjectName function instead of this. to get project name for this test